### PR TITLE
feat: Fixes packaging for release workflow

### PR DESCRIPTION
**Pull request by hernandazevedozup**
Fixes packaging for release workflow

### CODE REVIEW

File: android/beagle-scaffold-sample/build.gradle [hunk 1]
```diff
@@ -68,7 +68,7 @@ dependencies {
     kapt "br.com.zup.beagle:android-processor:$beagle_version"
 
     implementation project (":beagle-scaffold")
-
+//    implementation "br.com.zup.beagle:beagle-scaffold:1.0.0"
     implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9'
 
     testImplementation 'junit:junit:4.+'
```
Comment (botbeaglezup) [inline android/beagle-scaffold-sample/build.gradle:74]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold-sample/build.gradle#L74"></span>Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (junit:junit:4.+)</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold-sample/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -19,8 +19,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="br.com.zup.beagle.scaffold.sample">
 
-    <uses-permission android:name="android.permission.INTERNET" />
-
     <application
```
Comment (botbeaglezup) [inline android/beagle-scaffold-sample/src/main/AndroidManifest.xml:22]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold-sample/src/main/AndroidManifest.xml#L22"></span>On SDK version 23 and up, your app data will be automatically backed up and restored on app install. Consider adding the attribute <code>android:fullBackupContent</code> to specify an <code>@xml</code> resource which configures which files to backup. More info: https://developer.android.com/training/backup/autosyncapi.html</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/src/debug/AndroidManifest.xml [hunk 3]
```diff
@@ -19,7 +19,6 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
     package="br.com.zup.beagle.scaffold">
-    <uses-permission android:name="android.permission.INTERNET" />
     <application
         android:usesCleartextTraffic="true"
```
Comment (botbeaglezup) [inline android/beagle-scaffold/src/debug/AndroidManifest.xml:23]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold/src/debug/AndroidManifest.xml#L23"></span>Attribute <code>usesCleartextTraffic</code> is only used in API level 23 and higher (current min is 19)</td>
    </tr>
  </tbody>
</table>

**Comment by botbeaglezup** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          12 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-defaults-sample/build.gradle#L26">android/beagle-defaults-sample/build.gradle#L26</a> - Old buildToolsVersion 30.0.2; recommended version is 30.0.3 or later</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-defaults-sample/build.gradle#L64">android/beagle-defaults-sample/build.gradle#L64</a> - A newer version of com.google.android.material:material than 1.2.1 is available: 1.3.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-defaults-sample/build.gradle#L74">android/beagle-defaults-sample/build.gradle#L74</a> - Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (junit:junit:4.+)</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-defaults/build.gradle#L27">android/beagle-defaults/build.gradle#L27</a> - Old buildToolsVersion 30.0.2; recommended version is 30.0.3 or later</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-defaults/build.gradle#L57">android/beagle-defaults/build.gradle#L57</a> - A newer version of com.google.android.material:material than 1.2.1 is available: 1.3.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-defaults/build.gradle#L63">android/beagle-defaults/build.gradle#L63</a> - A newer version of junit:junit than 4.13 is available: 4.13.1</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold-sample/build.gradle#L26">android/beagle-scaffold-sample/build.gradle#L26</a> - Old buildToolsVersion 30.0.2; recommended version is 30.0.3 or later</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold-sample/build.gradle#L64">android/beagle-scaffold-sample/build.gradle#L64</a> - A newer version of com.google.android.material:material than 1.2.1 is available: 1.3.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold/build.gradle#L27">android/beagle-scaffold/build.gradle#L27</a> - Old buildToolsVersion 30.0.2; recommended version is 30.0.3 or later</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold/build.gradle#L66">android/beagle-scaffold/build.gradle#L66</a> - A newer version of com.google.android.material:material than 1.2.1 is available: 1.3.0</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold/build.gradle#L73">android/beagle-scaffold/build.gradle#L73</a> - A newer version of junit:junit than 4.13 is available: 4.13.1</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false"><a href="https://github.com/ZupIT/beagle-helpers/blob/77eccff0437a338495a621b9a2f62a08be308dd9/android/beagle-scaffold/build.gradle#L87">android/beagle-scaffold/build.gradle#L87</a> - A newer version of androidx.test.ext:junit than 1.1.3-alpha02 is available: 1.1.3-alpha03</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_android">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: android/android-test/build.gradle [hunk 4]
```diff
@@ -57,7 +57,7 @@ dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.4.20"
     implementation "com.facebook.yoga.android:yoga-layout:1.16.0"
     implementation "com.facebook.soloader:soloader:0.9.0"
-    implementation "junit:junit:4.13.1"
+    implementation "junit:junit:4.+"
```
Comment (botbeaglezup) [inline android/android-test/build.gradle:60]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/cfd139838c2a590f85e480dd265c24be5820ae93/android/android-test/build.gradle#L60"></span>Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (junit:junit:4.+)</td>
    </tr>
  </tbody>
</table>

File: android/beagle-defaults/build.gradle [hunk 5]
```diff
@@ -70,11 +70,11 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
     testImplementation "org.robolectric:robolectric:4.4"
-    testImplementation "junit:junit:4.13.1"
+    testImplementation 'junit:junit:4.+'
```
Comment (botbeaglezup) [inline android/beagle-defaults/build.gradle:73]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/cfd139838c2a590f85e480dd265c24be5820ae93/android/beagle-defaults/build.gradle#L73"></span>Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (junit:junit:4.+)</td>
    </tr>
  </tbody>
</table>

File: android/beagle-scaffold/build.gradle [hunk 6]
```diff
@@ -80,11 +81,11 @@ dependencies {
     androidTestImplementation 'androidx.test.ext:junit:1.1.2'
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.3.0'
     testImplementation "org.robolectric:robolectric:4.4"
-    testImplementation "junit:junit:4.13.1"
+    testImplementation 'junit:junit:4.+'
```
Comment (botbeaglezup) [inline android/beagle-scaffold/build.gradle:84]: <table data-meta="generated_by_android">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-helpers/blob/cfd139838c2a590f85e480dd265c24be5820ae93/android/beagle-scaffold/build.gradle#L84"></span>Avoid using + in version numbers; can lead to unpredictable and unrepeatable builds (junit:junit:4.+)</td>
    </tr>
  </tbody>
</table>

**Review by carlossteinzup — APPROVED** [review]

**Pull request merged**